rinex/src/algorithm/filters/decim.rs

```diff
@@ -33,22 +33,24 @@ pub trait Decimate {
     /// Header sampling interval (if any) is automatically updated.
     /// ```
     /// use rinex::prelude::*;
-    /// use rinex::processing::*;
+    /// use rinex::preprocessing::*; // Decimate
```

Comment:
My personal preference would be to import the trait directly instead of `*`, but it's fine if you like glob imports.